debian/control

```diff
@@ -2,7 +2,7 @@ Source: debathena-pyhesiodfs
 Section: debathena/net
 Priority: extra
 Maintainer: Debathena Project <debathena@mit.edu>
-Build-Depends: cdbs, debhelper, config-package-dev, python-support, python (>= 2.6~)
+Build-Depends: cdbs, debhelper, dh-systemd (>= 1.5), config-package-dev, python-support, python (>= 2.6~)
```

Comment:
I see. That would allow `dh-systemd` Build-Depend to be dropped on Trusty and earlier.